src/hardware/opl.h

```diff
@@ -152,47 +150,52 @@ class OPL: public Module_base {
 	IO_ReadHandleObject ReadHandler[3];
 	IO_WriteHandleObject WriteHandler[3];
 
-	//Mode we're running in
+	// Mode we're running in
 	Mode mode;
-	//Last selected address in the chip for the different modes
+	// Last selected address in the chip for the different modes
 	union {
 		uint32_t normal;
 		uint8_t dual[2];
 	} reg;
+
 	struct {
 		bool active;
 		uint8_t index;
 		uint8_t lvol;
 		uint8_t rvol;
 		bool mixer;
 	} ctrl;
-	void CacheWrite( uint32_t reg, uint8_t val );
-	void DualWrite( uint8_t index, uint8_t reg, uint8_t val );
-	void CtrlWrite( uint8_t val );
+
+	void CacheWrite(uint32_t reg, uint8_t val);
+	void DualWrite(uint8_t index, uint8_t reg, uint8_t val);
+	void CtrlWrite(uint8_t val);
+
 	uint8_t CtrlRead(void);
 
-	public:
+public:
 	static OPL_Mode oplmode;
 	mixer_channel_t mixerChan;
-	uint32_t lastUsed;				//Ticks when adlib was last used to turn of mixing after a few second
+
+	uint32_t lastUsed; // Ticks when adlib was last used to turn of mixing
+	                   // after a few second
 
 	RegisterCache cache;
-	Capture* capture;
-	Chip	chip[2];
+	Capture *capture;
+	Chip chip[2];
 
 	opl3_chip oplchip = {};
-	uint8_t newm = 0;
+	uint8_t newm      = 0;
 
```

Comment:
Might as well toss in defaults for all the members.
Reviewing commit-by-commit (please ignore if a later commit takes care of this :slightly_smiling_face: )